lang/semgrep-grammars/src/semgrep-typescript/common/semgrep-ext.js

```diff
@@ -39,6 +39,34 @@ module.exports = {
       $.optional_parameter,
     ),
 
+    class_body: $ => seq(
+      '{',
+      repeat(choice(
```

Comment:
would have been better to have an intermediate `class_element` rule in the original grammar so
we would not have to repeat so much from the original grammar.

Reply:
An issue is that if tomorrow the original grammar change class_body and add some constructs in it,
we have no way to statically check that we should also update our class_body :(
if instead we had an intermediate class_element, then could just extend the previous class_element
and use a `$.previous` like we do for the other rules.